### PR TITLE
23 CRUD for QueryStructureType

### PR DESCRIPTION
**Pull request by mwfrost**
This just adds the view for QueryStructureType, which already had serializers and factories.

### CODE REVIEW

File: chemreg/compound/tests/test_views.py [hunk 1]
```diff
@@ -171,3 +172,56 @@ def test_detail_view(self):
...
+        )
+        self.paginated_url = f"{self.list_url}?size=5&page=2"
+
+    def test_create_view(self):
+        new = {
+            "name": "new-querystructuretype",
+            "label": "a new query structure type",
+            "short_description": "described here",
+            "long_description": "described at greater length here",
+        }
```
Comment (mcovalt) [inline chemreg/compound/tests/test_views.py]: Can we use `QueryStructureTypeJSONFactory` for these?

Comment (mwfrost) [reply chemreg/compound/tests/test_views.py]: Switched.

File: chemreg/compound/models.py [hunk 2]
```diff
@@ -95,6 +95,9 @@ class QueryStructureType(CommonInfo):
         help_text="Query structure type long description",
     )
 
+    class Meta:  # without this pytest will throw a `UnorderedObjectListWarning`
+        ordering = ["created_at"]
+
```
Comment (mcovalt) [inline chemreg/compound/models.py]: This can be removed. PR #80 applies global ordering via the `CommonInfo` model.

**Comment by mcovalt** [comment]
This works as expected. The warnings from DRF regarding non-ordered responses will be resolved with PR #80. That makes all models ordered by the primary key.

**Pull request merged**